src/btchip_transaction.c

```diff
@@ -260,6 +260,16 @@ void transaction_parse(unsigned char parseMode) {
 
                     if (btchip_context_D.usingOverwinter ||
                         TRUSTED_INPUT_OVERWINTER) {
+                        if ((btchip_read_u32(btchip_context_D.transactionVersion, 0, 0) ^ (1<<31)) == 0x05) {
```

Comment:
Nit: since the version is the lowest 30 bits, maybe more clear/robust to do `& 0x3fffffff` instead of `^ (1<<31)`.